src/Ztm.Data.Entity/Contexts/Main/Input.cs

```diff
@@ -41,5 +41,35 @@ public int CompareTo(Input other)
 
             return 0;
         }
+
+        public override bool Equals(Object other)
+        {
+            if (other == null)
+            {
+                return false;
+            }
+
+            if (GetType() != other.GetType())
+            {
+                return false;
+            }
+
+            return CompareTo((Input)other) == 0;
+        }
+
+        public override int GetHashCode()
+        {
+            int hash = 0;
+
+            hash ^= (TransactionHash != null) ? TransactionHash.GetHashCode() : 0;
+            hash ^= (int)Index;
+            hash ^= (OutputHash != null) ? OutputHash.GetHashCode() : 0;
```

Comment:
We don't need this lines and below due to this class represent an input on a transaction. So just `TransactionHash` and `Index` should be enough.

Reply:
Make sense